cogment_verse/web/web_app/ndarray.proto

```diff
@@ -34,4 +34,5 @@ message Array {
   repeated double double_data = 5;
   repeated sint32 int32_data = 6;
   repeated sint64 int64_data = 7;
+  repeated uint32 uint32_data = 8;
```

Comment:
I did not even added this... not sure how it was added.

Reply:
I updated the gitignore to reflect the move from `web/` to `web/web_app/`. However, should I also just remove the file entirely? Instead of having a previously tracked but now untracked file that is not even matching the latest `cogment_verse/specs/ndarray.proto` file?

Reply:
Or just leave the updated file here.